extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -773,6 +785,20 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * This function is called if the ad response contains a creative size header
+   * indicating the size of the creative. It provides an opportunity to resize
+   * the creative, if desired, before it is rendered.
+   *
+   * To be implemented by network.
+   *
+   * @param {!Array<string>} size An array containing two elements, the first
```

Comment:
could this be changed to two params, width and height? it seems like everyone would need to be splitting the array into width and height anyway if it is passed in as an array